app/common/state/messageBoxState.js

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const tabState = require('./tabState')
+const {makeImmutable} = require('./immutableUtil')
+
+const messageBoxDetail = 'messageBoxDetail'
```

Comment:
does this need to be added to `docs/state.md`? does it need to be cleaned  from tabs on shutdown?